doc/user_guide/user_guide.md

```diff
@@ -25,82 +24,39 @@ within the scope of the project.
-| ssl-cert-path                |   [x]   | [x]  | Optional                                               |
-| [no_]use-ssl-cert-validation |   [x]   | [x]  | Optional boolean, defaults to True                     |
-| ssl-client-cert-path         |   [x]   |      | Optional                                               |
-| ssl-client-private-key       |   [x]   |      | Optional                                               |
-| develop                      |   [x]   | [x]  | Optional, if True, causes re-generation of the scripts |
-| verbose                      |   [x]   | [x]  | Optional, if True produces verbose output              |
-
-### TLS/SSL options
-
-The `--ssl-cert-path` is needed if the TLS/SSL certificate is not in the OS truststore.
-Generally speaking, this certificate is a list of trusted CA. It is needed for the server's certificate
-validation by the client.
-The option `--use-ssl-cert-validation`is the default, it can be disabled with `--no-use-ssl-cert-validation`.
-One needs to exercise caution when turning the certificate validation off as it potentially lowers the security of the
-Database connection.
-The "server" certificate described above shall not be confused with the client's own certificate.
-In some cases, this certificate may be requested by a server. The client certificate may or may not include
-the private key. In the latter case, the key may be provided as a separate file.
-
-### AWS Connection Object
-  - Create an Exasol connection object with AWS credentials that has 
-AWS Sagemaker Execution permission. The connection will encapsulate the address of the AWS S3 bucket where the exported data will be stored. 
+The deployment includes the installation of the Script Language Container (SLC) and several
+scripts. The SLC is a way to install the required programming language and necessary dependencies
+in the Exasol Database so that UDF scripts can be executed. The version of the installed SLC must
+match the version of the Sagemaker Extension Package. See [the latest release](https://github.com/exasol/sagemaker-extension/releases) on Github.
+
+For information about the available options common to all Exasol extensions please refer to the
+[documentation](https://github.com/exasol/python-extension-common/blob/0.8.0/doc/user_guide/user-guide.md) 
+in the Exasol Python Extension Common package. 
+In addition, this extension provides the following installation options:
+
+| Option name         | Default | Comment                                        |
+|:--------------------|:-------:|:-----------------------------------------------|
+| [no-]deploy-slc     |  True   | Install SLC as part of the deployment          |
+| [no-]deploy-scripts |  True   | Install scripts as part of the deployment      |
+| [no-]to-print       |  False  | Print SQL statements instead of executing them |
+| [no-]develop        |  False  | Re-generate the scripts                        | 
```

Comment:
this is internal

Reply:
I didn't understand the comment. Do you mean the last two options - `to_print` and `develop` - should not be exposed in the CLI, just the API?

Reply:
the --develop is an internal CLI